sound/soc/sof/intel/apl.c

```diff
@@ -18,13 +18,22 @@
 #include "../sof-priv.h"
```

Comment:
@ranj063 A bit offtopic but the commit message look a bit weird, you are wrapping lines at 50 columns. 
wrap at 74 columns would be the usual style, right.